glib/tests/derive.rs

```diff
@@ -0,0 +1,33 @@
+use std::marker::PhantomData;
+
+use glib::{Downgrade, Object};
+
+#[test]
+fn derive_downgrade() {
```

Comment:
Maybe add some tests here that this actually derives something meaningful and can be used, not just that it compiles :)

Also don't we have derive tests for this inside `glib-macros/tests` already?

Reply:
I have no idea. I simply moved this test out of the clone file. Don't know why it was there in the first place...